pepper/order_transform/generate.py

```diff
@@ -30,9 +30,14 @@ def hash_pedersen(left, right):
 	H = Point.from_x(FQ(16540640123574156134436876038791482806971768689494387082833631921987005038935))
 	return G * left + H * right
 
+#transform_int128 transform a 256 bit value into two 128 bit values by cutting the hash 
+# into two substrings and representing these substrings as ints with 39 digits 
+def transform_int128(hash):
+	s = hex(hash)
+	return [str(int(s[:34],0)).zfill(39),str(int(s[34:],16)).zfill(39)]
```

Comment:
Just a small question (no need to change), but why do we have to zero fill the ints?